assets/i18n/en.json

```diff
@@ -34,22 +34,22 @@
         "changelogLabel": "Changelog"
     },
     "latestCommitCard": {
-        "loadingLabel": "Loading",
+        "loadingLabel": "Loading...",
         "timeagoLabel": "{time} ago",
         "patcherLabel": "Patcher: ",
         "managerLabel": "Manager: ",
-        "updateButton": "Update Manager"
+        "updateButton": "Update"
```

Comment:
removing "Manager" will make this button ambiguous as to what it does for the user. I don't like this change